Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveHM.cpp

```diff
@@ -138,11 +142,18 @@ void FiniteDifferenceSolver::MacroscopicEvolveHMCartesian(
         amrex::Real const * const AMREX_RESTRICT coefs_y = m_stencil_coefs_y.dataPtr();
         amrex::Real const * const AMREX_RESTRICT coefs_z = m_stencil_coefs_z.dataPtr();
 
+        int lev = 0;
+        const auto dx = warpx.Geom(lev).CellSizeArray();
```

Comment:
this is just my own semantic preference. Since `dx` contains three components `dx`, `dy` and `dz`, naming the array as `dx` is a bit odd to me. Maybe `d_cell`?

Reply:
:) yes -- I agree that "dx" has "dx,dy,dz" for 3D and that its weird :)

But `dx=CellSizeArray()` is consistent with AMReX usage (https://amrex-codes.github.io/amrex/docs_html/Basics.html#realbox-and-geometry), so its used that way in warpx too.
So for consistency, we can choose to have the same usage.

Reply:
Going with the AMReX convention, I am sticking with dx.